runtime/mainnet/src/config/governance.rs

```diff
@@ -0,0 +1,344 @@
+use frame_support::{
+	parameter_types,
+	traits::{EitherOfDiverse, NeverEnsureOrigin},
+	weights::Weight,
+};
+use frame_system::EnsureRoot;
+use pallet_collective::EnsureProportionAtLeast;
+use parachains_common::BlockNumber;
+use pop_runtime_common::DAYS;
+
+use crate::{
+	AccountId, Council, Runtime, RuntimeBlockWeights, RuntimeCall, RuntimeEvent, RuntimeOrigin,
+};
+
+type UnanimousCouncilVote = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 1>,
+>;
+
+type AtLeastThreeFourthsOfCouncil = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 4>,
+>;
+
+parameter_types! {
+	pub CouncilMotionDuration: BlockNumber = 7 * DAYS;
+	pub const CouncilMaxProposals: u32 = 100;
+	pub const CouncilMaxMembers: u32 = 100;
+	pub MaxProposalWeight: Weight = sp_runtime::Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block;
+}
+
+pub type CouncilCollective = pallet_collective::Instance1;
+impl pallet_collective::Config<CouncilCollective> for Runtime {
+	type Consideration = ();
+	type DefaultVote = pallet_collective::PrimeDefaultVote;
```

Comment:
Should this not be changed? E.g. https://github.com/polkadot-fellows/runtimes/blob/1a8ac56dc901a523ee2dad5f696989a57b94f038/system-parachains/collectives/collectives-polkadot/src/lib.rs#L567C40-L567C76

Reply:
By using `PrimeDefaultVote` and not setting a prime member we give councilors the ability to abstain. Which otherwise is not an option.

Reply:
Does abstaining inevitably mean that a single vote by prime member pushes something through? Would've expected the threshold still to be honoured.

Reply:
I've changed `DefaultVote` to use `MoreThanMajorityThenPrimeDefaultVote` in: [5cdced0](https://github.com/r0gue-io/pop-node/pull/448/commits/5cdced084955b84588d1432b27165da337df889a)

Abstaining will become an aye if:
- More than half of seats have voted aye or Prime has voted aye.

The threshold of ayes does still need to be met for a proposal to be approved.

Reply:
Updated these tests at: [d86f49f](https://github.com/r0gue-io/pop-node/pull/448/commits/d86f49fead6fd39f3c4069979211601d8e141d58)